sdk/python/tests/compiler/testdata/artifact_location.yaml

```diff
@@ -20,7 +20,7 @@ spec:
   params:
   - name: bucket
   - name: namespace
-  - name: secret_name
+  - name: secret-name
```

Comment:
Thanks @ckadner, yes capital letters and underscore characters are support for parameter name, but not for results value, and we already coverted it in results value, this will lead to inputs and outputs inconsistency. It was tested out for my big data passing cases, and also reproduced for watson cases.